docs/components/checkbox.md

```diff
@@ -55,6 +55,26 @@ Use `v-model` to bind the `checked` state of the underlying `<input />`. The `v-
 </KCheckbox>
 ```
 
+### label
+
+Will place label text to the right of the checkbox. Can also be [slotted](#slots).
+
+```html
+<KCheckbox v-model="labelPropChecked1" label="Label Example" />
```

Comment:
nit: Usually we simplify the variables used in the examples 
```suggestion
<KCheckbox v-model="checked" label="Label Example" />
```

Reply:
Done